flow/project.py

```diff
@@ -1190,153 +1075,64 @@ def get(k, m):
-                            yield list(row)
-                    else:
-                        row.insert(1, None)
-                        yield list(row)
-                else:
-                    yield row
+                status['parameters'] = OrderedDict()
+                for i, k in enumerate(parameters):
+                    v = shorten(str(self._alias(get(k, sp))))
+                    parameters_length[i] = max(parameters_length[i], len(v))
+                    status['parameters'][k] = v
 
             for status in statuses.values():
-                rows_status.extend(_format_status(status))
-
-            rows_operations = []
-            header_operations = [self._tr(self._alias(s))
-                                 for s in ('job_id', 'operation', 'eligible', 'cluster_status')]
+                _add_parameters(status)
 
-            def _fmt_status_operations(status):
-                for name, doc in status['operations'].items():
-                    if not _select_op(doc):
-                        continue
-
-                    yield [
-                        status['job_id'],
-                        _bold(name) if (pretty and doc['eligible']) else name,
-                        'Y' if doc['eligible'] else 'N',
-                        _FMT_SCHEDULER_STATUS[doc['scheduler_status']]]
-
-            for status in statuses.values():
-                rows_operations.append(list(_fmt_status_operations(status)))
-
-        # Actually display information
-        if detailed:
-            print(('\n' if overview else '') + "# Detailed View:", file=file)
-            status_table = tabulate.tabulate(rows_status, headers=header_detailed)
-            if expand:  # Present labels and operations in two separate tables.
-                print("\n## Labels:", file=file)
-                print(status_table, file=file)
-                print("\n## Operations:", file=file)
-                rows_operations = [row for rs in rows_operations for row in rs]  # flatten list
-                print(tabulate.tabulate(rows_operations, headers=header_operations), file=file)
-            elif unroll:
-                print(status_table, file=file)
-            else:       # Present labels and operations in a combined 'compact' view.
-                # We need to split the labels table into individual lines
-                # to combine them with the operations lines.
-                status_table_lines = iter(status_table.splitlines())
-
-                # The first two lines are the table header.
-                print(next(status_table_lines), file=file)
-                print(next(status_table_lines), file=file)
-
-                def _print_unicode(value):
-                    "Python 2/3 compatibility layer."
-                    if six.PY2:
-                        print(value.encode('utf-8'), file=file)
-                    else:
-                        print(value, file=file)
+        if template is None:
```

Comment:
I think this section (lines 1087-1095) should come right after line 1042, because it helps the reader understand what is happening next "gathering of the context for these templates".